### PR TITLE
Add dependency

### PR DESCRIPTION
**Pull request by sergio-chainguard**
Fixes:

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [ ] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [ ] REQUIRED - The version of the package is still receiving security updates
- [ ] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

#### For new version streams

- [ ] The upstream project actually supports multiple concurrent versions.
- [ ] Any subpackages include the version string in their package name (e.g. `name: ${{package.name}}-compat`)
- [ ] The package (and subpackages) `provides:` logical unversioned forms of the package (e.g. `nodejs`, `nodejs-lts`)

#### For security-related PRs

- [ ] The security fix is recorded in the [advisories](https://github.com/wolfi-dev/advisories) repo

#### For version bump PRs

- [ ] The `epoch` field is reset to 0

#### For PRs that add patches

- [ ] Patch source is documented

### CODE REVIEW

**Review by imjasonh — APPROVED** [review]

**Comment by hectorj2f** [comment]
We missed to bump the epoch here!

File: kubeflow-katib.yaml [hunk 1]
```diff
@@ -52,7 +49,8 @@ subpackages:
           output: katib-${{range.key}}
           ldflags: -w -X main.version=${{package.version}}
           subpackage: true
-          deps: golang.org/x/net@v0.17.0 github.com/docker/distribution@v2.8.2 github.com/docker/docker@v24.0.9 google.golang.org/grpc@v1.56.3 golang.org/x/crypto@v0.17.0
+          deps: golang.org/x/net@v0.23.0 github.com/docker/distribution@v2.8.2 github.com/docker/docker@v26.0.2+incompatible google.golang.org/grpc@v1.56.3 golang.org/x/crypto@v0.17.0 google.golang.org/protobuf@v1.33.0
+          replaces: golang.org/x/net=golang.org/x/net@v0.23.0
```
Comment (hectorj2f) [inline kubeflow-katib.yaml:53]: We should use go/bump here instead of go/build for bumping dependencies.

File: kubeflow-katib.yaml [hunk 2]
```diff
@@ -14,6 +14,7 @@ environment:
       - ca-certificates-bundle
       - gfortran
       - go
+      - hdf5-dev
```
Comment (hectorj2f) [inline kubeflow-katib.yaml:17]: We missed to bump the epoch too.

File: kubeflow-katib.yaml [hunk 3]
```diff
@@ -33,10 +34,6 @@ pipeline:
       expected-commit: fc6beec8358f1c4a50d86de63c5906fce2eba373
       tag: v${{package.version}}
 
-  - uses: go/bump
```
Comment (hectorj2f) [inline kubeflow-katib.yaml:36]: This is still needed.

**Pull request merged**